src/client-ducky/src/components/SearchBar.js

```diff
@@ -17,41 +59,63 @@ const SearchBar = ({url, method, fullscreen}) => {
     }
 
     function handleClick() {
+        debugger;
         console.log("searched clicked");
+        history.push('/s');
     }
 
     useEffect(() => {
         api.get('parking-sizes/').then(res => {
-            setParkingSpots(res.data)
+            if (res.data.length > 0) {
+                setParkingSpots(res.data)
+            } else {
+                setParkingSpots(fakeData);
```

Comment:
why did we keep this fake data thing? I really think it's best to show a message saying nothing is there...

Reply:
I made this fake data to see how the page was rendering because the database kept returning empty objects

Reply:
That's probably because you didn't have very many if not any spots in your db that were either valid or created, so I can show you how it's done. It was working on mine, and changing it back to this is just a lot more messy than getting our db populated.